### PR TITLE
Fixed #197 -- Fixed ugettext_lazy() deprecation warnings for Django 3.0.

### LINKED ISSUE #197: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated
# Actual Behavior

When running my django tests with -Wa after upgrading to Django 3.0.1 (from 2.2.*), I get a lot of deprecation warnings:

```
./venv3/bin/python3.6 ./manage.py test --settings=return_code.settings_dev
./venv3/lib/python3.6/site-packages/request/utils.py:10: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
  (100, _('Continue')),
./venv3/lib/python3.6/site-packages/request/utils.py:11: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
  (101, _('Switching Protocols')),
./venv3/lib/python3.6/site-packages/request/utils.py:12: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
  (102, _('Processing (WebDAV)')),
./venv3/lib/python3.6/site-packages/request/utils.py:15: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
  (200, _('OK')),
./venv3/lib/python3.6/site-packages/request/utils.py:16: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
  (201, _('Created')),
./venv3/lib/python3.6/site-packages/request/utils.py:17: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
  (202, _('Accepted')),
./venv3/lib/python3.6/site-packages/request/utils.py:18: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
  (203, _('Non-Authoritative Information')),
./venv3/lib/python3.6/site-packages/request/utils.py:19: RemovedInDjango40Warning: django.utils.translation.ugettext_lazy() is deprecated in favor of django.utils.translation.gettext_lazy().
  (204, _('No Content')),
(...)
```

# Expected Behavior:
- I should not get those warnings

# Analysis

As the warning already states, what the issue is, we should simply replace the import in `./request/utils.py`

```

from django.utils.translation import ugettext_lazy as _
```
by

```
from django.utils.translation import ugettext_lazy as _
```

I'm not sure how this would affect backwards compatibility.

### PR DESCRIPTION
**Pull request by jwedel**
#197
- Fixed deprecaation warning by replacing gettext import in utils.py
- Added IntelliJ files to .gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,7 @@ dist/
 *.sw[pon]
 .coverage
 .tox
+
+# IntelliJ project and module files
+.idea
+*.iml
```
Comment (felixxm) [inline .gitignore]: Please revert this unrelated change.

Comment (jwedel) [reply .gitignore]: Those are IntelliJ project files added to gitignore. Why is this an issue?

Comment (felixxm) [reply .gitignore]: I don't see any reason to add IDE-specific files to `.gitignore`.

Comment (jwedel) [reply .gitignore]: Ok, you’re probably right. I will add those to my global gitignore and revert. Thanks for your feedback!

**Comment by jwedel** [comment]
@felixxm I think this should be OK now.

**Comment by felixxm** [comment]
@jwedel Thanks :+1:
